routes/tags.js

```diff
@@ -0,0 +1,12 @@
+const express = require("express");
+
+const { addTag, deleteTag, getAllTags } = require("../controllers/tags");
+const { validTagBody } = require("../middlewares/validators/tags");
+
+const router = express.Router();
+
+router.post("/", validTagBody, addTag);
+router.delete("/:tagid", deleteTag);
```

Comment:
we don't want everyone to add or delete tags, can you authenticate for the superuser role here

Reply:
sure, will add it